fhir-install/src/main/docker/ibm-fhir-schematool/DEVELOPMENT.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title:  IBM FHIR Server - Schema Tool - Development
+date:   2020-11-11
+permalink: /ibm-fhir-server-schema-tool-development/
+---
+
+# Development
```

Comment:
Is this file picked up and used some place?   I might have expected this type of info in our wiki rather than with the docker artifacts.

Reply:
It's development only - no wiki, just trying to keep these notes inline. I thought it might end up as a guide at some point.